AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -342,7 +343,7 @@ - (void)applyEnabledState:(BOOL)isEnabled {
     [self emptyLogBufferFiles];
     [self removeAnalyzerFile];
     [self.plCrashReporter purgePendingCrashReport];
-    [[MSSessionContext sharedInstance] clearSessionHistory];
+    [self clearContextHistoryAndKeepCurrentSession:NO];
```

Comment:
We should keep the most recent one or store the current session when Crashes is enabled.
I feel this way is more complicated because the Crashes should know it was disabled in the same life cycle.
Instead of differentiate it between disabled and then enabled vs Crashes started and then enabled, we can simply keep the most recent one here.